.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Package
+      run: |
+        python setup.py sdist bdist_wheel
+    - name: Publish to PyPI
+      uses: pypa/gh-action-pypi-publish@master
+      with:
+        user: __token__
+        password: ${{ secrets.pypi_password }}
```

Comment:
Will require you setting this in the repo settings under secrets.

Further information around generating this: https://github.com/pypa/gh-action-pypi-publish#usage

Reply:
Will Get it done over the next few days.Thanks.

Reply:
No rush - but let me know if there is anything that can be done to help.